test/from-draft3/to-draft4/schema.json

```diff
@@ -0,0 +1,73 @@
+[
+    {
+        "title": "Simple matching schema",
+        "from": {
+            "$schema": "https://json-schema.org/draft-03/schema"
+        },
+        "to": {
+            "$schema": "https://json-schema.org/draft-04/schema"
+        }
+    },
+    {
+        "title": "Simple non matching schema",
+        "from": {
+            "$schema": "https://json-schema.org/draft/2020-12/schema"
+        },
+        "to": {
+            "$schema": "https://json-schema.org/draft/2020-12/schema"
+        }
+    },
+    {
+        "title": "Nested matching schema",
+        "from": {
+            "$schema": "https://json-schema.org/draft-03/schema",
+            "$defs": {
+                "foo": {
+                    "$schema": "https://json-schema.org/draft-03/schema"
+                }
+            }
+        },
+        "to": {
+            "$schema": "https://json-schema.org/draft-04/schema",
+            "$defs": {
+                "foo": {
+                    "$schema": "https://json-schema.org/draft-04/schema"
```

Comment:
I don't think you should be transforming this one, as `$defs` is not defined in Draft 3